modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -0,0 +1,101 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-kubernetes-nmstate-operator-CLI_{context}"]
+= Installing the Kubernetes NMState Operator using the CLI
+Instead of using the web console, you can install the Kubernetes NMState Operator by using the {product-title} CLI (oc).
```

Comment:
Wondering if you have any thoughts about omitting the introductory clause: 

You can install the Kubernetes NMState Operator by using the {product-title} CLI (oc) instead of using the web console.

Or simply: 

You can install the Kubernetes NMState Operator by using the {product-title} CLI (oc).

Or

You can also install the Kubernetes NMState Operator by using the {product-title} CLI (oc).